packaging/leapp-repository.spec

```diff
@@ -227,6 +227,7 @@ done;
 %dir %{repositorydir}
 %dir %{custom_repositorydir}
 %dir %{leapp_python_sitelib}/leapp/cli/commands
+%exclude %{leapp_python_sitelib}/leapp/cli/commands/tests
```

Comment:
@vinzenz @fernflower  this will result in possible problems as rpmbuild will report some "installed" files are not part of any rpm. Instead of this, put the removal of the directory into the `%install` section as we do for other tests directories. Currently it's not visible as the directory doesn't exist yet.

IOW, when the `%exclude` cmd is used, the files should be present in another RPM. So the removal from the installation dirs is the better way.
Putting this on the line 189 does the thing:
```
rm -rf %{buildroot}%{leapp_python_sitelib}/leapp/cli/commands/tests
```

Reply:
updated, thanks!